packages/api/src/typeDefs/shipping.graphql

```diff
@@ -0,0 +1,299 @@
+"""
+Shipping Simulation item input.
+"""
+input IShippingItem {
+  """
+  ShippingItem ID / Sku.
+  """
+  id: String!
+  """
+  Number of items.
+  """
+  quantity: Int!
+  """
+  Seller responsible for the ShippingItem.
+  """
+  seller: String!
+}
+
+"""
+Shipping Simulation information.
+"""
+type ShippingData {
+  """
+  List of LogisticsItem.
+  """
+  items: [LogisticsItem]
+  """
+  List of LogisticsInfo.
+  """
+  logisticsInfo: [LogisticsInfo]
+  """
+  List of MessageInfo.
+  """
+  messages: [MessageInfo]
+  """
+  Address information.
+  """
+  address: Address
+}
+
+"""
+Shipping Simulation Logistic Item.
+"""
+type LogisticsItem {
+  """
+  LogisticsItem ID / Sku.
+  """
+  id: String
+  requestIndex: Int
+  """
+  Number of items.
+  """
+  quantity: Int
+  """
+  Seller responsible for the ShippingItem.
+  """
+  seller: String
+  """
+  List of Sellers.
+  """
+  sellerChain: [String]
+  """
+  LogisticsItem tax.
+  """
+  tax: Int
+  """
+  Next date in which price is scheduled to change. If there is no scheduled change, this will be set a year in the future from current time.
+  """
+  priceValidUntil: String
+  """
+  LogisticsItem price.
+  """
+  price: Int
+  """
+  LogisticsItem listPrice.
+  """
+  listPrice: Int
+  """
+  LogisticsItem rewardValue.
+  """
+  rewardValue: Int
+  """
+  LogisticsItem sellingPrice.
+  """
+  sellingPrice: Int
+  """
+  LogisticsItem measurementUnit.
+  """
+  measurementUnit: String
+  """
+  LogisticsItem unitMultiplier.
+  """
+  unitMultiplier: Int
+  """
+  LogisticsItem availability.
+  """
+  availability: String
+}
+
+type LogisticsInfo {
+  """
+  LogisticsInfo itemIndex.
+  """
+  itemIndex: String
+  """
+  LogisticsInfo selectedSla.
+  """
+  selectedSla: String
+  """
+  List of LogisticsInfo ShippingSLA.
+  """
+  slas: [ShippingSLA]
+}
+
+type ShippingSLA {
+  """
+  ShippingSLA id.
+  """
+  id: String
+  """
+  ShippingSLA name.
+  """
+  name: String
+  """
+  ShippingSLA price.
+  """
+  price: Float
+  """
+  ShippingSLA shipping estimate.
+  """
+  shippingEstimate: String
+  """
+  ShippingSLA friendly shipping estimate.
+  """
+  friendlyShippingEstimate: String
```

Comment:
If you do change this to `localizedShippingEstimate`, this will also need to change :)

Oh! And how about using this for the docs?

```
  """
  ShippingSLA localized shipping estimate. 
  
  Note: this will always return a localized string for locale `en-US`.
  """

```

Reply:
Good suggestion.  Thanks 🎉 
addressed [here](https://github.com/vtex/faststore/pull/1440/commits/9c3675835d343adf50825925de9b0027d79b89bf).